content/courses/advanced-ada/parts/resource_management/memory_management.rst

```diff
@@ -6,9 +6,209 @@ Memory Management
 Maximum allocation size and alignment
 -------------------------------------
 
-.. todo::
+We've seen details about the :ada:`Size` and :ada:`Object_Size` attributes in the section about :ref:`data representation <Adv_Ada_Data_Representation_Sizes>`. :ref:`Later on <Adv_Ada_Storage_Size_Attribute>`, we also mentioned the :ada:`Storage_Size` attribute.
 
-    Complete section!
+In this section, we expand our discussion on sizes and talk about the :ada:`Max_Size_In_Storage_Elements` and the :ada:`Max_Alignment_For_Allocation` attributes. These attributes return values that are important in the allocation of :ref:`memory subpools <Adv_Ada_Memory_Pools>` via the :ada:`Allocate` procedure from the :ada:`System.Storage_Pools.Subpools` package:
+
```

Comment:
No just subpools though. Both that and System.Storage_Pools as well.

Reply:
Very good point, thanks for the hint!